ApeWorX/ape
# feat: start ephemeral development geth when no HTTP provider running

### What I did

fixes: #204 

I can open a concise PR for the first pass of `ape test` as soon as this gets merged in.
I have tested these changes with my `ape test` work. The tests run super duper fast!

### How I did it

Geth PoA chain using genesis file to spawn accounts from the test configured mnemonic
Deletes data dir on shutdown... Acts ephemerally.

### How to verify it

Run a simple script.
Notice that it first spawns up a local PoA blockchain and then runs the script.
Make sure to use an account that is based off the test-mnemonic!

### Checklist
